lib/oli_web/router.ex

```diff
@@ -218,7 +218,7 @@ defmodule OliWeb.Router do
 
     # Project
     put "/:project_id", ProjectController, :update
-    delete "/:project_id", ProjectController, :delete
+    post "/:project_id", ProjectController, :delete
```

Comment:
It makes more sense to have the verb here be DELETE, but I think I understand why you changed it, so that the title can be passed along as a body param.  The title can be passed along via a query param, though, which would allow the verb stay DELETE

Reply:
switch back to delete verb